### PR TITLE
CVH Integration!

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**

### CODE REVIEW

**Review by lspdrz — APPROVED** [review]

File: src/api/Session.ts [hunk 1]
```diff
@@ -26,9 +25,15 @@ export async function loginWithCredentials(
...
   });
   const body = await response.json();
+
   if (response.status !== 201) {
     Store.dispatch(setSessionStatus("inactive"));
-    throw body;
+    throw response;
   }
-  await initializeSession(body);
+  // TODO: we should improve this
```
Comment (mpekala23) [inline src/api/Session.ts:33]: might it be worth to link to an issue on `connect-api-nest`? These two are similar, or you could edit / open a new issue:
- https://github.com/Ameelio/connect-api-nest/issues/70
- https://github.com/Ameelio/connect-api-nest/issues/67

File: src/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -69,18 +69,22 @@ function Dashboard({
...
-          <PDFDownloadButton
-            calls={callsToday(calls)}
-            facility={facility}
-            canViewDetails={false}
-          />,
-        ]}
+        extra={
+          [
+            // TODO: add this back, complete info is not being populated
+            // https://github.com/Ameelio/connect-doc-client/issues/57
```
Comment (mpekala23) [inline src/pages/Dashboard/index.tsx:75]: 🔥🔥🔥

File: src/pages/LiveCall/VideoChat.tsx [hunk 3]
```diff
@@ -107,22 +105,14 @@ const VideoChat: React.FC<Props> = React.memo(
...
-            socket.emit(
-              "authenticate",
-              {
-                type: "monitor",
-                id,
-                token,
-              },
-              resolve
-            );
+            socket.emit("authenticate", authInfo, resolve);
```
Comment (mpekala23) [inline src/pages/LiveCall/VideoChat.tsx:108]: nice

**Review by mpekala23 — APPROVED** [review]
one quick nit (linking to a GH issue) but looks great!

**Pull request merged**